gulpfile.js

```diff
@@ -37,15 +37,15 @@ gulp.task('clean:tests', false, (done) => {
 });
 
 gulp.task('clean', 'Clean all', ['clean:dist', 'clean:docs', 'clean:tests'], (done) => {
```

Comment:
in theory we should have indentation 4 spaces but tsjs doesn't seem to autofix js files for that rule :( , don't do anything for now with this